.travis.yml

```diff
@@ -19,6 +18,9 @@ before_install:
   # Start tesseract server.
   - "tesseract-master/bin/tesseract >>tesseract.txt 2>&1 &"
 
+  # The npm on travis is quite old, update it
+  - npm install -g npm@'>=1.4.3'
```

Comment:
In older versions of node, npm does not understand the `^` character in versions. So we need a minimum of npm 1.4.3.